src/test/java/de/mediathekview/mserver/testhelper/AssertFilm.java

```diff
@@ -0,0 +1,55 @@
+package de.mediathekview.mserver.testhelper;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mlib.daten.Sender;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.hamcrest.Matchers;
+import static org.junit.Assert.assertThat;
+
+public final class AssertFilm {
+  private AssertFilm() {}
+
+  public static void assertEquals(final Film aActualFilm,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 13 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)